workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -28,6 +28,19 @@ import kotlin.reflect.KType
  *
  * All methods have default no-op implementations.
  *
+ * ## Additional Methods
+ *
+ * There are 3 more methods in this interface:
+ *
+ *  1. [onSessionStarted] - called when a new [WorkflowSession] is created the first time a
+ *     workflow is rendered with the [CoroutineScope] for that session.
+ *  1. [onRenderingSkipped] - called when the rendering is short-circuited because no state
+ *     change was detected as part of the action cascade(s) and the
+ *     [RuntimeConfig.RENDER_ONLY_WHEN_STATE_CHANGES] is enabled.
+ *  1. [onRenderingUpdated] - called when a [RenderingAndSnapshot] is passed to the [StateFlow]
+ *     returned by [renderWorkflowIn]. It is passed the [RenderingAndSnapshot] which could be
+ *     decorated.
```

Comment:
Stale kdoc, there is only one new method now.

Reply:
Will update.